code/game/objects/items/weapons/RCD.dm

```diff
@@ -69,7 +69,8 @@ RCD
 	var/deconwindowdelay = 50
 	var/deconairlockdelay = 50
 
-	var/no_ammo_message = ""
+	var/no_ammo_message = "<span class='warning'>The \'Low Ammo\' light on \
```

Comment:
why would you even cut this line, it's not even long, it actually makes it worst to read than the entire line together.